### PR TITLE
Mark all Swift-specific changes in API with comments. NFC

### PR DESCRIPTION
**Pull request by adrian-prantl**
This is to facilitate identifying divergences and thus encourage
upstreaming them and/or make merge conflict resolutions more obvious.

### CODE REVIEW

**Comment by JDevlieghere** [comment]
The diff makes it hard to verify that we have everything, but LGTM.

File: lldb/source/API/CMakeLists.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
+## BEGIN SWIFT
 set(LLVM_NO_RTTI 1)
```
Comment (dcci) [inline lldb/source/API/CMakeLists.txt:2]: Is there a reason why this is needed?

File: lldb/source/API/SBTarget.cpp [hunk 2]
```diff
@@ -71,6 +69,11 @@
 #include "llvm/Support/PrettyStackTrace.h"
 #include "llvm/Support/Regex.h"
 
+// BEGIN SWIFT
+#include "lldb/Breakpoint/Breakpoint.h"
```
Comment (dcci) [inline lldb/source/API/SBTarget.cpp]: I wonder if these can just go away to reduce the diff.

File: lldb/source/API/SBTarget.cpp [hunk 3]
```diff
@@ -1078,11 +1081,14 @@ SBTarget::BreakpointCreateForException(lldb::LanguageType language,
                      (lldb::LanguageType, bool, bool), language, catch_bp,
                      throw_bp);
 
+  // BEGIN SWIFT
```
Comment (dcci) [inline lldb/source/API/SBTarget.cpp:1083]: What's exactly the diff here?

File: lldb/source/API/SBThread.cpp [hunk 4]
```diff
@@ -412,19 +412,22 @@ size_t SBThread::GetStopDescription(char *dst, size_t dst_len) {
...
+  // END SWIFT
 }
 
+// BEGIN SWIFT
 SBValue SBThread::GetStopErrorValue() {
   bool is_swift_error_value = false;
   SBValue return_value = GetStopReturnOrErrorValue(is_swift_error_value);
   if (!is_swift_error_value)
-    return SBValue();
+    rturn SBValue();
```
Comment (dcci) [inline lldb/source/API/SBThread.cpp]: typo?

File: lldb/source/API/SystemInitializerFull.cpp [hunk 5]
```diff
@@ -119,11 +117,16 @@
 #include "lldb/Host/windows/windows.h"
 #endif
 
+// BEGIN SWIFT
+#include "lldb/Symbol/SwiftASTContext.h"
+#include "Plugins/InstrumentationRuntime/SwiftRuntimeReporting/SwiftRuntimeReporting.h"
+
 #if defined(__APPLE__) || defined(__linux__) || defined(_WIN32)
```
Comment (dcci) [inline lldb/source/API/SystemInitializerFull.cpp]: This `#ifdef` can be dropped.

**Review by dcci — COMMENTED** [review]
Thanks!

**Pull request merged**